core/templates/services/image-local-storage.service.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Oppia Authors. All Rights Reserved.
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
Undo these changes in all the places:
1. A PR should have minimal changes.
2. The year represents the year in which the file was created.

Reply:
I'm not sure about this.
I have been updating copyright year in all of my PRs and maintainers sometimes ask me to do it.
![Screenshot from 2021-03-20 13-53-19](https://user-images.githubusercontent.com/24855641/111863905-2c7cd900-8984-11eb-8447-63db53bf1b30.png)

![Screenshot from 2021-03-20 13-56-42](https://user-images.githubusercontent.com/24855641/111863910-34d51400-8984-11eb-9b56-4e12a11eb249.png)

Reply:
Done